static/js/components/dashboard/CourseGrade.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import React from 'react';
+
+import { Course } from '../../flow/programTypes';
+import { asPercent } from '../../util/util';
+import {
+  STATUS_PASSED,
+  STATUS_NOT_PASSED,
+  STATUS_ENROLLED_NOT_VERIFIED,
+  STATUS_VERIFIED_NOT_COMPLETED,
+} from '../../constants';
+
+export default class CourseGrade extends React.Component {
+  props: {
+    course: Course,
+    now: Object,
+  };
+
+  render() {
+    const { course } = this.props;
+    let firstRun = {};
+    if (course.runs.length > 0) {
+      firstRun = course.runs[0];
+    }
+
+    let percent = null;
+    let description = null;
+
+    if (firstRun.grade !== undefined) {
```

Comment:
Oh I see, in that case we'd just end up putting an empty `{percent}` and `{description}` on the page. nvm, carry on.